llvm/test/CodeGen/AMDGPU/sdwa-peephole-cndmask-sext.ll

```diff
@@ -0,0 +1,21 @@
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx942 < %s | FileCheck %s
+; XFAIL: *
+
+; FIXME The sext modifier is turned into a neg modifier in the asm output
+
+define i32 @test_select_on_sext_sdwa(i8 %x, i32 %y, i1 %cond)  {
```

Comment:
Put issue number in the function name.

Can you merge the test into one of the existing sdwa tests

Reply:
Merged the pre-si-peephole-sdwa mir into an existing mir test with checks that reflect the current expectation (no sdwa conversion) instead of using xfail.

Reply:
The MIR test isn't really a substitute, particularly given that there are whole pipeline interactions with the final emission. Should bring back the IR test too